apps/a11y-tests/src/axe-sarif-converter/axe-extension.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { CheckConfiguration, RuleConfiguration } from './ruleresults';
+
+declare module 'axe-core/axe' {
```

Comment:
This file's contents should probably go in (relative to the repo root) `/typings/axe-core/axe/index.d.ts` (or possibly `/typings/axe-core/axe.d.ts`).

Then in this project's tsconfig.json compilerOptions, add:
```
"typeRoots": ["../../typings", "node_modules/@types"]
```

There may be a bit more tweaking of settings and/or syntax required to make this work--let me know if you run into trouble.

Reply:
Code in `apps/a11y-tests/src/axe-sarif-converter` is a temporary copy and will be removed once official converter is published. In the meantime I will try to make it compile against OUFR's tslint/tsc options, but would prefer not to make re-organizing/refactoring changes.